### PR TITLE
added code to calculate p_val from chi_squared_stat

### PR DESCRIPTION
**Pull request by faisalmlalani**
Modified the function that computes a X^2 statistic to also calculate and return a p_value. Other functions are modified to receive this new input and present it in the final output.

### CODE REVIEW

File: outlierdetect/outlierdetect.py [hunk 1]
```diff
@@ -105,49 +105,54 @@ def compute_outlier_scores(self, frequencies):
...
             rng = frequencies[list(frequencies.keys())[0]].keys()
             outlier_scores = {}
             expected_frequencies = {}
+            p_values = {}
             for agg_unit in list(frequencies.keys()):
                 summed_freq = self._sum_frequencies(agg_unit, frequencies)
                 expected_frequencies[agg_unit] = summed_freq
                 if(sum(summed_freq.values()) == 0):
                     outlier_scores[agg_unit] = 0
+                    p_values[agg_unit] = -1
```
Comment (bderenzi) [inline outlierdetect/outlierdetect.py]: Should this be 1?

File: outlierdetect/outlierdetect.py [hunk 2]
```diff
@@ -105,49 +105,54 @@ def compute_outlier_scores(self, frequencies):
...
                 expected: a dictionary giving the expected frequencies, e.g.,
                     {'y' : 13.2, 'n' : 17.2}
                 actual: a dictionary in the same format as the expected dictionary
                     (with the same range) giving the actual distribution.
-            
+
             Returns:
-                the X^2 statistic for the actual frequencies, given the expected frequencies.
+                the X^2 statistic for the actual frequencies, given the expected frequencies,
+                the p-value of the result.
```
Comment (bderenzi) [inline outlierdetect/outlierdetect.py]: maybe add the word "and" since this is not a list

File: outlierdetect/outlierdetect.py [hunk 3]
```diff
@@ -105,49 +105,54 @@ def compute_outlier_scores(self, frequencies):
                     (number of times aggregation unit reported value).
 
             Returns:
-                - dictionary mapping (aggregation unit) -> (MMA outlier score for aggregation unit).
-                - dictionary mapping (aggregation unit) ->  return from _sum_frequencies: (a dictionary mapping (value) -> (number of times all aggregation units apart from agg_unit reported this value)).
+                dictionary mapping (aggregation unit) -> (MMA outlier score for aggregation unit).
```
Comment (bderenzi) [inline outlierdetect/outlierdetect.py]: Double check this? I think we added one return value in the tuple instead of removing?

**Review by bderenzi — CHANGES_REQUESTED** [review]
I assume we also need to change the example file since the API changed?

File: example.py [hunk 4]
```diff
@@ -28,18 +28,20 @@ def print_scores(scores):
...
     data = pd.read_csv(DATA_FILE)  # Uncomment to load as pandas.DataFrame.
     # data = mlab.csv2rec(DATA_FILE)  # Uncomment to load as numpy.recarray.
 
     # Compute SVA outlier scores.
-    (sva_scores, _) = outlierdetect.run_sva(data, 'interviewer_id', ['cough', 'fever'])
-    print("SVA outlier scores")
-    print_scores(sva_scores)
+    #(sva_scores, _) = outlierdetect.run_sva(data, 'interviewer_id', ['cough', 'fever'])
+    #print("SVA outlier scores")
+    #print_scores(sva_scores)
```
Comment (bderenzi) [inline example.py]: what happened here?

**Review by bderenzi — CHANGES_REQUESTED** [review]
Great, looks much better. Minor question in there about the SVA example code. I think we still want that to work even though we're not actively extending it.

File: example.py (hunk 4 above)
Comment (faisalmlalani) [reply example.py]: I think I just commented it a while ago when testing stuff out. Uncommented.

**Review by bderenzi — APPROVED** [review]

**Pull request merged**